File: cpp/for loop/factorial.cpp
/*Find the factorial of a given number*/

#include <iostream>
using namespace std;

int main() {
    int n, i;
    long fact=1;

    cout<<"Enter any number: ";
    cin>>n;

    for(i = 1; i <= n; i++) {
        
        fact=fact*i;
        
    }
        cout<<fact<<endl;
    
    return 0;
}

File: java/for loop/factorial.java
/*Find the factorial of a given number*/

import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        int n, i;
        long fact=1;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any number:");
        n = sc.nextInt();
        
        for(i = 1; i <= n; i++) {
        fact=fact*i;
    }
        System.out.println("factorial is " + fact);
    }
}